LEAF_Request_Portal/js/parallelProcessing.js

```diff
@@ -46,17 +46,20 @@ function parallelProcessing(recordID, orgChartPath, CSRFToken)
         $.ajax({
             type: 'GET',
             url: 'api/?a=form/'+recordID+'/workflow/indicator/assigned',
-            success: function(obj) {
-                indicatorObject = obj;
-                for (var i = 0; i < indicatorObject.length; i++) {
-                    $(document.createElement('option'))
-                        .attr('value', indicatorObject[i].indicatorID)
-                        .html(indicatorObject[i].name)
-                        .appendTo($("select#indicator_selector"));
+            success: function(arr) {
+                indicatorObject = arr;
+                if (indicatorObject !== null) {
+                    for (let i = 0; i < indicatorObject.length; i++) {
+                        $(document.createElement('option'))
+                            .attr('value', indicatorObject[i].indicatorID)
+                            .html(indicatorObject[i].name)
+                            .appendTo($("select#indicator_selector"));
+                    }
                 }
-                if(obj.length == 0) {
+                else {
+                    let failResponse = '<span style="font-size: 120%">Error: The form/workflow must contain a field of type "orgchart group" or "orgchart employee" to begin Parallel Processing.</span>';
```

Comment:
Since you're in here...  This is almost what I'm looking for.

let failResponse = 'Error: The form/workflow must contain a field of type "orgchart group" or "orgchart employee" to begin Parallel Processing.';

let responseHTML = '<span style="font-size: 120%">' + failResponse + '</span>';

The idea is to keep types clean.  "failResponse" is data.  And should be able to be changed without changing the "code".  "responseHTML" is presentation-code.  It will be handled with Vue in a future iteration.  And the Vue function will take the "failResponse" as an argument and return formatted html.

Reply:
Since this is outside of the ticket specifically, I'll bring the "Code Grooming ticket in to this sprint and you can put your hours there.  Please make sure you hit this at every point in the files you touch for the ticket.